mkdocs-website/docs/en/learn/guides/file-associations.md

```
@@ -0,0 +1,139 @@
+| description | Description shown in file properties                             | Windows  |
+| iconName    | Name of the icon file (without extension) in the build folder    | All      |
+| role        | Application's role for this file type (e.g., `Editor`, `Viewer`) | macOS    |
+
+## Listening for File Open Events
+
+To handle file open events in your application, you can listen for the `events.Common.ApplicationOpenedWithFile` event:
+
+```go
+func main() {
+    app := application.New(application.Options{
+		Name: "MyApp", 
```

Markdownlint

Column: 1
Hard tabs

(MD010, no-hard-tabs)